packages/repository/package.json

```diff
@@ -20,6 +20,7 @@
   "author": "IBM",
   "license": "MIT",
   "devDependencies": {
+    "@loopback/core": "^4.0.0-alpha.12",
```

Comment:
This is suspicious. Having `repository` depend on the `core` is a bad thing. In our case, `core` is mostly for http rest apis.

Reply:
I've updated it so `core` is only a test/dev dependency now.